src/server/cards/intrigue/Swindler.js

```diff
@@ -13,6 +13,7 @@ export default class Swindler extends Card {
       if (!card) {
         return;
       }
+      player.gameLog(`${other.name} reveals ${card.title}`);
```

Comment:
maybe these should be put in `selectCard` and `selectOption` instead of in the cards themselves

Reply:
Maybe include an additional parameter like, logSelection because we don't want to log everything like when we choose a copper with moneylender or something trivial like that. Or maybe all that detail should be logged.